Feature/dev

magrathj opened the pull request:
Two commits to address Issues: #31 and #26 . Added documentation for #26 and added feature to saveDataFlatfile function for #31 .

magrathj committed:
type: feature

Adding feature to enable changing of the flat's filename. Addressing issue @31

"export with a diferent file name #31"

magrathj committed:
type: doc

Wrote documentation on how to use Google sheets as a storage type. Addresses issue "writing to STORAGE_TYPES$GOOGLE_SHEETS works with non empty googlesheet #26 "

daattali commented on R/shinyform.R:
```diff
@@ -84,16 +84,23 @@ loadData <- function(storage) {
-    ),
-    ".csv"
-  )
+  if(!is.null(formInfo$storage$filename)){
```
This `saveDataFlatfile` function only has two parameters: data and storage. `formInfo` is not a parameter in this function. It can't be used here.

magrathj replied:
Yeah sorry should have just been storage$filename

daattali commented on R/shinyform.R:
```diff
@@ -84,16 +84,23 @@ loadData <- function(storage) {
+      formInfo$storage$filename,
+      ".csv"
+    )
+  }else{
```
Please add whitespace before and after brackets (in general, follow the tidyverse style guide https://style.tidyverse.org)

magrathj replied:
Cool, will keep to this style guide from now on

daattali commented:
Can you submit a separate PR for each issue please? It makes it cleaner and easier to follow each feature. Thanks for your help!

magrathj committed:
type: fix

removed formInfo and added whitespace to adhere to tidy style spacing requirements https://style.tidyverse.org/syntax.html#spacing
